CODE_OF_CONDUCT.md

```diff
@@ -1,139 +1,147 @@
+
+## Encouraged behaviors <a name="encouraged-behaviors"></a>
+
+While acknowledging differences in social norms, we all strive to meet our community's expectations for positive behavior. We also understand that our words and actions may be interpreted differently than we intend based on culture, background, or native language.
+
+With these considerations in mind, we agree to behave mindfully toward each other and act in ways that center our shared values, including:
+
+  1. Respecting the **purpose of our community**, our activities, and our ways of gathering.
+  2. Engaging **kindly and honestly** with others.
+  3. Respecting **different viewpoints** and experiences.
+  4. **Taking responsibility** for our actions and contributions.
+  5. Gracefully giving and accepting **constructive feedback**.
+  6. Committing to **repairing harm** when it occurs.
+  7. Behaving in other ways that promote and sustain the **well-being of our community**.
+
+## Restricted behaviors <a name="restricted-behaviors"></a>
+
+We agree to restrict the following behaviors in our community. Instances, threats, and promotion of these behaviors are violations of this Code of Conduct.
+
+  1. **Harassment.** Violating explicitly expressed boundaries or engaging in unnecessary personal attention after any clear request to stop.
+  2. **Character attacks.** Making insulting, demeaning, or pejorative comments directed at a community member or group of people.
+  3. **Stereotyping or discrimination.** Characterizing anyone’s personality or behavior on the basis of immutable identities or traits.
+  4. **Sexualization.** Behaving in a way that would generally be considered inappropriately intimate in the context or purpose of the community.
+  5. **Violating confidentiality**. Sharing or acting on someone's personal or private information without their permission.
+  6. **Endangerment.** Causing, encouraging, or threatening violence or other harm toward any person or group.
+  7. Behaving in other ways that **threaten the well-being** of our community.
+
+### Other restrictions <a name="other-restrictions"></a>
+
+  1. **Misleading identity.** Impersonating someone else for any reason, or pretending to be someone else to evade enforcement actions.
+  2. **Misrepresenting project affiliation.** Speaking or acting in a way that implies an official affiliation with the BeeWare project, where one does not exist.
+  3. **Failing to credit sources.** Not properly crediting the sources of content you contribute.
+  4. **Promotional materials.** Sharing marketing or other commercial content in a way that is outside the norms of the community.
+  5. **Excessive communication.** Disrespecting the time and space of others by engaging in an unacceptable volume of communication.
+  6. **Unnecessary interactions.** Unnecessarily reacting to or involving yourself in interactions that do not require a response from you.
```

Comment:
I'm a little hesitant about this one. We don't want to discourage people from offering *informed* opinions when a subject is being discussed by others - especially when there's a power dynamic between the people discussing, and the person offering the suggestion. Suggestion:
```suggestion
  6. **Unhelpful communication.** Offering opinions without relevant experience in the topic being discussed, entering into an ongoing discussion without first gaining familiarity with the history of the topic, or making contributions that are off-topic or otherwise distracting.
```

Reply:
This is an excellent suggestion. I feel it is clearer and speaks more to the point I was trying to get across.